flybywiresim/a32nx
# fix: Update flight model with correct station locations and weights

Fixes #3430 

## Summary of Changes

Updated weight and balance section with data from real A320-200 aircraft (CEO and NEO are identical for w&b purposes). Changed the empty weight CG location so that CG is 25% MAC as designed by Airbus. The aircraft has 4 sections with a total of 162 pax in a single class of service (Economy) and 4 holds. Each location is correct per IATA AHM-560 locations and are based on real life aircraft TC-JUK (only valid for W&B). The reference points are correct per Airbus specifications as documented in issue 3430. The data has been crossed checked with the Airbus A320 Weight and Balance Manual (WBM).

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Make sure aircraft is loaded per max values specified for each station
2. Ensure that fuel is loaded appropriately
3. Ensure that TOW CG does not fall outside valid range of 16-40% MAC
4. Ensure ZFW CG does not fall outside of above range
5. Ensure Simbrief integration is loading payload correctly


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
